book/website/communication/research-objects.md

```diff
@@ -0,0 +1,37 @@
+(cm-ro)=
+# Research Object to capture the Research Life Cycle
+
+## Prerequisites / recommended skill level
+
+## Summary
+
+Research outcomes encompass publications, data, software, bibliographical material and any other resources (such as experimental protocols, standards) that can be potentially useful for conduting the research. 
+
+Research Objects allow to work open by design and share during the research process and not only the research outputs at the end.
+
+A Research Object can be implemented as a "Research Object Crate" that is a **machine readable "bundle"** of all research artefacts as a whole.
+
+```{figure} ../figures/research-object.jpg
+---
+name: orcid-ids
+alt: This image shows how research objects evolve and grow during collaboration process and how new research objects can be created.
+---
+Research Objects allow to work open by design and share during the research process and not only the research outputs at the end. _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+(cm-ro-motivation)=
+## Background & Motivation
+
+The main motivations are:
+- to organize and describe the resources, materials, and methods of the research while conducting the research and not only at the end when publishing the outcomes of the research;
+- to manage the entire research lifecycle, and allow the release and publication of results progressively, keeping track of versioning and change information;
+- to share your research materials with other scientists at discrete milestones of your investigation, and collaborate via a single information unit uniquely identified by an PID, possible a DOI;
+- to enhance the findability and accessibility of all the scientific outcomes through a single information unit associated with rich, machine-readable metadata;
+- to facilitate reproducibility and reuse of the scientific methods and results via access to resources, context and metadata, and the possibility of forking existing results;
+- to be recognized and cited (even of constituent parts) encouraging a faster release and publication of research materials;
+- to preserve results and prevent decay using provenance for partial rerun;
```

Comment:
I'm not sure what is meant with 'provenance for partial rerun'

Reply:
we've removed the term provenance for partial rerun

Reply:
yes. it is very confusing and we removed it.